apps/angular/src/app/features/anime/anime-page/anime-page.component.ts

```diff
@@ -0,0 +1,98 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { PageEvent } from '@angular/material/paginator';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { Anime } from '@js-camp/core/models/anime';
+import { AnimeSortField, AnimeParams } from '@js-camp/core/models/anime-params';
+import { AnimeStatus } from '@js-camp/core/models/anime-status';
+import { Pagination } from '@js-camp/core/models/pagination';
+import { BehaviorSubject, Observable, tap, map, debounceTime, switchMap, shareReplay } from 'rxjs';
+
+const defaultParams: AnimeParams = {
+	limit: 10,
+	page: 0,
+	sorting: { field: AnimeSortField.None, direction: 'asc' },
+	filters: {
+		type: [],
+		search: '',
+	},
+};
+
+const REQUEST_DEBOUNCE_TIME = 500;
+
+/** Anime list page. */
+@Component({
+	selector: 'camp-anime-page',
+	templateUrl: './anime-page.component.html',
+	styleUrls: ['./anime-page.component.css'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimePageComponent {
+
+	/** Columns in a table. */
+	protected readonly displayedColumns: readonly string[] = ['image', 'titleEnglish', 'titleJapanese', 'aired.start', 'type', 'status'];
+
+	/** Page size options. */
+	protected readonly pageSizeOptions: readonly number[] = [5, 10, 20];
+
+	/** Anime list. */
+	protected readonly animeList$: Observable<Pagination<Anime>>;
+
+	/** Anime is loading. */
+	protected readonly isLoading$ = new BehaviorSubject<boolean>(false);
```

Comment:
Generic is redundant where ts can infer type without a problem
```suggestion
	protected readonly isLoading$ = new BehaviorSubject(false);
```